functions.js

```diff
@@ -58,6 +58,20 @@ function mapContentLinks(languages) {
   }));
 }
 
+const contentFilters = require('./data/content_filters.json');
+function filterContents(languages) {
+    return languages.map(l => ({
+        ...l,
+        contents: l.contents.filter((c) => {
+           if (l.code in contentFilters && contentFilters[l.code].includes(c.code)) {
+               console.log('Filtered out lang:' + l.code + " content:" + c.code);
```

Comment:
I don't know if this is necessarily needed but ok

Reply:
Mainly the console.log

Reply:
Thanks for your note.  I was going for consistency with the rest of the code, where it logs to the console when it does something unusual.  I'm fine with taking it out though; it's basically a debug statement.